README.rst

```diff
@@ -64,15 +70,28 @@ key to download a request using Zyte API. Full list of parameters is provided in
 
 .. code-block:: python
 
-    yield scrapy.Request(
-        "http://books.toscrape.com/",
-        callback=self.parse,
-        meta={
-            "zyte_api": {
-                "browserHtml": True,
-                "geolocation": "US",
-                "javascript": True,
-                "echoData": {"something": True}
-            }
-        }
-    )
+   import scrapy
+
+   class TestSpider(scrapy.Spider):
+       name = 'test'
+       start_urls = ['http://books.toscrape.com/']
```

Comment:
We can remove this since `start_urls` isn't being used since `start_requests()` was overridden below.

Reply:
Yes, I removed it now.